static/app/views/replays/detail/ai/useFetchReplaySummary.tsx

```diff
@@ -96,6 +97,7 @@ export function useFetchReplaySummary(
   // summary data query and 2) we have the stale version of the summary data. The consuming
   // component will briefly show a completed state before the summary data query updates.
   const startSummaryRequestTime = useRef<number>(0);
+  const globalTimeoutRef = useRef<NodeJS.Timeout | null>(null);
```

Comment:
The type here should be `number` if the calls are `window.setTimeout` and `window.clearTimeout`. it's something about typescript picking the node types if it's underspecified.